src/hooks/useToggleColors.js

```diff
@@ -12,12 +12,12 @@ const Colors = {
   anon: {
     primary: COLORS.anon_primary,
     secondary: COLORS.anon_secondary,
-    text: COLORS.white
+    text: COLORS.almostBlack
   },
   signed: {
     primary: COLORS.signed_primary,
     secondary: COLORS.signed_secondary,
-    text: COLORS.white
+    text: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the color for the text in the `signed` object has been changed from `COLORS.white` to `COLORS.almostBlack`. Again, verify that this change doesn't negatively impact readability on any screen where it's applied.


<!-- This is an auto-generated comment by OSS CodeRabbit -->